### PR TITLE
2827 Show generic "shape" symbol for core shapes

### LINKED ISSUE #2827: Show generic "shape" symbol for core shapes
For generic shapes we show an upside-down question mark:
image

This is because we are trying to show a MilSymbol:
image

Instead, we should only try to show a MilSymbol where `_type` property is `MilSymRenderer`:
```
 "_type": "MilSymRenderer"
```

For `CoreRenderer`, we should show a generic `shape` icon (the `shapes` icon from FontAwesome).

Note: this will also resolve the issue of mistakenly trying to show health indicator for core shapes: https://github.com/serge-web/serge/pull/2814#issuecomment-1929291088

### PR DESCRIPTION
**Pull request by lebaphi**
Fixes #2827 

image

### CODE REVIEW

**Comment by IanMayo** [comment]
The symbol is great, thanks @lebaphi .

I wanted to test changing the force color, but I'm unable to get `save` the new color.

Also note, please. We don't need to show `health` for these core shapes.  Hmm, but maybe that's because some test data mistakenly contains `health` properties.

It would be more robust to only add the health bar for features of type `MilSymRenderer` (or whatever the constant is).

**Comment by IanMayo** [comment]
Ok @lebaphi the `save` issue is now under #2836, but could you please make the change to only show the health bar for `MilSymRenderer` features please?

**Comment by lebaphi** [comment]
> Ok @lebaphi the `save` issue is now under #2836, but could you please make the change to only show the health bar for `MilSymRenderer` features please?

okay @IanMayo

File: client/src/Components/local/atoms/mapping-panel/helpers/icon-renderer.tsx [hunk 1]
```diff
@@ -12,23 +16,24 @@ type IconRendererProps = {
...
+  const iconElm = useMemo(() => {
+    if (!feature) {
+      return
+    }
     const icon = new ms.Symbol(feature.properties?.sidc, { size: 20 })
     const health = feature.properties?.health
 
     const healthColor = calculateHealthColor(health)
 
-    if (iconRef.current) {
```
Comment (IanMayo) [inline client/src/Components/local/atoms/mapping-panel/helpers/icon-renderer.tsx:24]: Hey @lebaphi - I tried again and again to make this fix myself, I just couldn't manage the tsx constructs for it. 

Thanks!

**Review by IanMayo — APPROVED** [review]
Tested and working

**Pull request merged**